### DISCUSSION TITLE
User count mismatch on course dashboard and reports. would like to know how it works

### DISCUSSION THREAD

**Discussion by sachinb80** [Sunbird-Obsrv/Community#160] [Query] [answered]
Hello all,
We are using the 5.0 version of Sunbird
The user count on the course dashboard and the user count in reports are mismatched.
I want to how and why it is happening
Also, want to know how the user count data are fetched on the dashboard and in the report.
Want to know the workaround for user count.
If anyone has any idea regarding it please guide me as I need some assistance here
Sharing some screenshot of the dashboard and report for reference
![image (28)](https://github.com/Sunbird-Obsrv/Community/assets/109504832/61a9dc79-90e8-42cd-b0bd-a88a9c9405b6)
![image (27)](https://github.com/Sunbird-Obsrv/Community/assets/109504832/2cf41502-81dd-4d34-992e-4fa6a4b6fcd9)
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

**Comment by Venkatesh-Dayani23** [comment]
Hi Team,
We have checked all the druid servers and the ingestion tasks all are running fine without any errors.There is no lag and any failed events in the Kafka.Still the count is not matching with count visible in the portal and the report generated.Kindly let us know is there any chance to resync again.Can you also suggest us where to check to findout the root cause of the issue as we are not seeing any error in druid and pipeline.

**Reply by sowmya-dixit** [reply, marked as answer]
@sachinb80 @Venkatesh-Dayani23 
These reports are generated from Sunbird-Ed exhaust jobs, since there are no issues with druid and pipeline - Can you please post this query on Sunbird-Ed discussion forum for more details on how it works. 

Let us know if you face any issues with pipeline or Druid.

**Reply by sachinb80** [reply]
@sowmya-dixit Okay
